.github/workflows/tests.yml

```diff
@@ -22,15 +28,15 @@ jobs:
     name: Runs the tests
     steps:
       - name: Check out source code
-        uses: actions/checkout@v3.0.2
+        uses: actions/checkout@v3.3.0
```

Comment:
I am using the latest versions of all workflows because the old versions [generated deprecation warnings](https://github.com/Automattic/vip-go-mu-plugins-ext/actions/runs/3898704734/attempts/1)